### PR TITLE
Support Integrity Block v2

### PR DESCRIPTION
**Pull request by GrapeGreen**

### CODE REVIEW

File: packages/rollup-plugin-webbundle/README.md [hunk 1]
```diff
@@ -273,6 +282,10 @@ https://github.com/WICG/isolated-web-apps.
 
 ## Release Notes
 
+### v0.2.0
+
+- Add support for the v2 integrity block format.
```
Comment (sonkkeli) [inline packages/rollup-plugin-webbundle/README.md:287]: nit: can you add a very short 1 sentence description here too what that means?

File: packages/rollup-plugin-webbundle/README.md [hunk 2]
```diff
@@ -235,9 +236,17 @@ const pluginOptionsWithCustomSigningStrategy = {
 };
 ```
 
+### `integrityBlockSign.strategies`
+
+Type: `Array<ISigningStrategy>`
+
+Use this overload to sign a bundle with multiple keys or to specify a custom
+`webBundleId`.
+
```
Comment (sonkkeli) [inline packages/rollup-plugin-webbundle/README.md:245]: also similarly to the integrityBlockSign.strategy, please add an example with this type of signing

File: packages/rollup-plugin-webbundle/README.md (hunk 1 above)
Comment (sonkkeli) [reply packages/rollup-plugin-webbundle/README.md:287]: also both of these comments apply to the other readme too

**Review by sonkkeli — COMMENTED** [review]
LGTM apart from the documention nits

File: packages/rollup-plugin-webbundle/README.md (hunk 1 above)
Comment (GrapeGreen) [reply packages/rollup-plugin-webbundle/README.md]: Done!

File: packages/rollup-plugin-webbundle/README.md (hunk 2 above)
Comment (GrapeGreen) [reply packages/rollup-plugin-webbundle/README.md:265]: Done!

**Review by sonkkeli — APPROVED** [review]

**Pull request merged**